ansible/roles/ndh-app/tasks/prereqs.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: shell enable repos in redhat
+  ansible.builtin.shell: |
```

Comment:
better to use https://docs.ansible.com/ansible/latest/collections/ansible/builtin/yum_module.html